packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.component.tsx

```diff
@@ -182,170 +182,173 @@ function AllergyForm({ closeWorkspace, promptBeforeClosing, patientUuid }: Defau
-            <div className={styles.wrapper}>
-              <RadioButtonGroup
-                name="severity-of-worst-reaction"
-                onChange={(event) => setSeverityOfWorstReaction(event.toString())}
-                valueSelected={severityOfWorstReaction}
-              >
-                {severityLevels.map((severity, index) => (
-                  <RadioButton
-                    id={severity.toLowerCase() + 'Severity'}
-                    key={`severity-${index}`}
-                    labelText={severity}
-                    value={(() => {
-                      switch (severity.toLowerCase()) {
-                        case 'mild':
-                          return mildReactionUuid;
-                        case 'moderate':
-                          return moderateReactionUuid;
-                        case 'severe':
-                          return severeReactionUuid;
-                        default:
-                          return '';
-                      }
-                    })()}
+            </section>
+            <section className={styles.section}>
+              <h2 className={styles.sectionHeading}>{t('dateAndComments', 'Date and comments')}</h2>
+              <div className={styles.wrapper}>
+                <DatePicker
+                  id="onsetDate"
+                  dateFormat="d/m/Y"
+                  datePickerType="single"
+                  light={!isTablet}
+                  maxDate={new Date().toISOString()}
+                  onChange={([date]) => setOnsetDate(date)}
+                  value={onsetDate}
+                >
+                  <DatePickerInput
+                    id="onsetDateInput"
+                    placeholder="dd/mm/yyyy"
+                    labelText={t('dateOfFirstOnset', 'Date of first onset')}
                   />
-                ))}
-              </RadioButtonGroup>
-            </div>
-          </section>
-          <section className={styles.section}>
-            <h2 className={styles.sectionHeading}>{t('dateAndComments', 'Date and comments')}</h2>
-            <div className={styles.wrapper}>
-              <DatePicker
-                id="onsetDate"
-                dateFormat="d/m/Y"
-                datePickerType="single"
-                light={!isTablet}
-                maxDate={new Date().toISOString()}
-                onChange={([date]) => setOnsetDate(date)}
-                value={onsetDate}
-              >
-                <DatePickerInput
-                  id="onsetDateInput"
-                  placeholder="dd/mm/yyyy"
-                  labelText={t('dateOfFirstOnset', 'Date of first onset')}
+                </DatePicker>
+              </div>
+              <div className={styles.wrapper}>
+                <TextArea
+                  className={styles.textbox}
+                  cols={20}
+                  light={!isTablet}
```

Comment:
Presumably you're seeing a warning about this by now saying to use `Layer` instead

Reply:
Yeah, good shout. I'll address those in a separate PR.